jobs/library.sh

```diff
@@ -77,6 +77,10 @@ join_hyperv (){
     echo "devstackIP: $FIXED_IP"
     echo "branchName: $ZUUL_BRANCH"
     echo "buildFor: $ZUUL_PROJECT"
+    if [[ ! -z $IS_DEBUG_JOB ]] && [[ $IS_DEBUG_JOB = "yes" ]]; then
+        run_wsmancmd_with_retry $1 $2 $3 '"powershell Write-Host Calling create-environment with devstackIP='$FIXED_IP' branchName='$ZUUL_BRANCH' buildFor='$ZUUL_PROJECT' >>\\'$FIXED_IP'\openstack\logs\create-environment-'$1'.log 2>&1"'
```

Comment:
these lines are not ok. You don't do debug output long before doing the command.